wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession.swift

```diff
@@ -1710,7 +1710,12 @@ extension ZMUserSession {
                 journal: journal,
                 sessionManager: sessionManager
             ),
-            AppVersionMigration_4_3_0(coreCryptoProvider: coreCryptoProvider)
+            AppVersionMigration_4_3_0(coreCryptoProvider: coreCryptoProvider),
+            AppVersionMigration_4_8_0(
+                contextProvider: coreDataStack,
+                conversationLocalStore: clientSessionComponent?.conversationLocalStore,
+                protobufMessageProcessor: clientSessionComponent?.conversationProtobufMessageProcessor
+            )
```

Comment:
thought: personally I find it misleading to have a migration that doesn't actually do anything. I suggested in a previous comment to make the prepare the migration template that can be reused, but I wouldn't actually create one until it's needed.

Reply:
How do we currently discover there are new message types? Is it a manual process? Perhaps there is a place we can leave a comment that will easily be found by the person who nexts introduces a new message type.